File: 2024-2/0684/unidad06.d_Web_04/SoftPub/SoftPubPersonaControler/DAO/SeccionDAO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Runtime.Remoting.Metadata.W3cXsd2001;
using System.Text;
using System.Threading.Tasks;
using SoftPubPersonaModel;

namespace SoftPubPersonaControler.DAO
{
    public interface SeccionDAO
    {        
        int insertar(Seccion seccion);

        int modificar(Seccion seccion);

        int eliminar(Seccion seccion);

        BindingList<Object> listarTodos();

        Seccion obtenerPorId(string idSeccion);        
    }
}

File: 2024-2/0682/unidad07.e - Programación distribuida - RMI - con BD/SoftPub_Back-end/ServidorRMI/SoftPubPersonaDA/src/pe/edu/pucp/softpub/persona/dao/SeccionDAO.java
package pe.edu.pucp.softpub.persona.dao;

import java.util.ArrayList;
import pe.edu.pucp.softpub.persona.model.Seccion;

/**
 *
 * Autor: andres
 */
public interface SeccionDAO {

    public Integer insertar(Seccion seccion);

    public Integer modificar(Seccion seccion);

    public Integer eliminar(Seccion seccion);

    public ArrayList<Seccion> listarTodos();
    
    public Seccion obtenerPorId(String idSeccion);
}
